test/unit_tests/test_test_results.py

```diff
@@ -44,6 +44,26 @@ def test_test_results(self):
         finally:
             shutil.rmtree(rootdir)
 
+    def test_test_results_detail(self):
+        results = {'test1.xml': (5, 1, 3), 'test2.xml': (7, 2, 4)}
+        try:
+            oldstdout = sys.stdout
+            sys.stdout = StringIO()
+            rootdir = tempfile.mkdtemp()
+            for filename in ['test1.xml', 'test2.xml', 'foo.bar']:
```

Comment:
Why should this test use more then one file? It is about the detailed output and not about correct discovery or parsing of result files.